cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -40,12 +51,53 @@ function($, Backbone, _, gettext, BaseView, BaseModal, Feedback, StringUtils, Mo
 
         show: function() {
             BaseModal.prototype.show.apply(this, [false]);
-            Feedback.prototype.inFocus.apply(this, [this.options.modalWindowClass]);
         },
 
         hide: function() {
+            if (this.moveXBlockListView) {
+                this.moveXBlockListView.remove();
+            }
+            if (this.moveXBlockBreadcrumbView) {
+                this.moveXBlockBreadcrumbView.remove();
+            }
             BaseModal.prototype.hide.apply(this);
             Feedback.prototype.outFocus.apply(this);
+        },
+
+        focusModal: function() {
+            Feedback.prototype.inFocus.apply(this, [this.options.modalWindowClass]);
+        },
+
+        fetchCourseOutline: function() {
+            var self = this;
+            $.when(
+                $.ajax({
+                    url: this.outlineURL,
+                    contentType: 'application/json',
+                    dataType: 'json',
+                    type: 'GET'
+                }),
+                $.ajax({
+                    url: this.XBlockAncestorInfoUrl,
+                    contentType: 'application/json',
+                    dataType: 'json',
+                    type: 'GET'
+                })
+            ).then(function(outlineResponse, ancestorResponse) {
```

Comment:
In case of any error in response to AJAX call, [global ajax error handler](https://github.com/edx/edx-platform/blob/master/cms/static/cms/js/main.js#L27) will be triggered and show the bottom error notification.